djstripe/models/billing.py

```diff
@@ -155,14 +162,15 @@ class Coupon(StripeModel):
         default=0,
         help_text="Number of times this coupon has been applied to a customer.",
     )
-    # valid = models.BooleanField(editable=False)
+    valid = models.BooleanField(
+        help_text="Taking account of the above properties, whether this coupon can still be applied to a customer.",
+        null=True,
+        blank=True,
+    )
```

Comment:
I'm not comfortable merging this as it does not get automatically updated via webhooks.